specification/mongocluster/DocumentDB.MongoCluster.Management/MongoCluster.tsp

```diff
@@ -33,13 +34,12 @@ interface MongoClusters {
     LroHeaders = Azure.Core.Foundations.RetryAfterHeader
   >;
   /** Updates an existing mongo cluster. The request body can contain one to many of the properties present in the normal mongo cluster definition. */
-  update is ArmCustomPatchAsync<
-    MongoCluster,
-    Azure.ResourceManager.Foundations.ResourceUpdateModel<
-      MongoCluster,
-      MongoClusterProperties
-    >
-  >;
+  @sharedRoute
+  updateOld is ArmResourcePatchAsync<MongoCluster, MongoClusterProperties>;
```

Comment:
since we are updating the current version, no need to introduce updateOld

Reply:
fixed